File: official/docs/node/carrier_accounts/create_carrier_account.js
const Easypost = require('@easypost/api');
const api = new Easypost('EASYPOST_API_KEY');

const carrierAccount = new api.CarrierAccount({
  type: 'DhlEcsAccount',
  description: 'CA Location DHL eCommerce Solutions Account',
  credentials: {
    client_id: '123456',
    client_secret: '123abc',
    distribution_center: 'USLAX1',
    pickup_id: '123456',
  },
  test_credentials: {
    client_id: '123456',
    client_secret: '123abc',
    distribution_center: 'USLAX1',
    pickup_id: '123456',
  },
});

carrierAccount.save().then(console.log);

File: official/docs/go/carrier_accounts/create_carrier_account.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	carrierAccount, err := client.CreateCarrierAccount(
        &easypost.CarrierAccount{
            Type:        "DhlEcsAccount",
            Description: "CA Location DHL eCommerce Solutions Account",
            Credentials: map[string]string{
                "client_id": "123456",
                "client_secret": "123abc",
                "distribution_center": "USLAX1",
                "pickup_id": "123456"
            },
            TestCredentials: map[string]string{
                "client_id": "123456",
                "client_secret": "123abc",
                "distribution_center": "USLAX1",
                "pickup_id": "123456"
            },
        },
    )

	fmt.Println(carrierAccount) 
}
